rancher2/resource_rancher2_global_role.go

```diff
@@ -94,13 +94,16 @@ func resourceRancher2GlobalRoleUpdate(d *schema.ResourceData, meta interface{})
 			return resource.NonRetryableError(err)
 		}
 
+		log.Println("updated inherited_cluster_roles", toArrayString(d.Get("inherited_cluster_roles").([]interface{})))
```

Comment:
I think that we can remove this now. None of the other fields have a log statement, so I don't think we should have a special case for this one.

Reply:
Absolutely, it was used to troubleshoot the update issue and wasn't meant to be committed.